tests/test_test_tools/test_test_cases.py

```diff
@@ -169,6 +169,16 @@ def test_disable_aea_logging(self):
         """Call logging disable."""
         self.disable_aea_logging()
 
+    def test_run_interaction(self):
+        """Test run_interaction"""
+        process = self.run_interaction()
+        process.terminate()
```

Comment:
This is not great at all, I don't know what "interact" is even supposed to be, it doesn't seem to exist as far as I can tell.

https://github.com/valory-xyz/open-aea/blob/50024dca077bc331971e9f300f8824bf79bac7c7/aea/test_tools/test_cases.py#L428-L437